migrations/20210730183235_add_email_change_confirmed.up.sql

```diff
@@ -0,0 +1,15 @@
+-- adds email_change_confirmed
+
+ALTER TABLE auth.users
+ADD COLUMN IF NOT EXISTS email_change_token_current varchar(255) null DEFAULT '', 
+ADD COLUMN IF NOT EXISTS email_change_confirm_status smallint DEFAULT 0 CHECK (email_change_confirm_status >= 0 AND email_change_confirm_status <= 2);
+
+DO $$
+BEGIN
+  IF NOT EXISTS(SELECT *
```

Comment:
Should this be `IF EXISTS`?

Reply:
ah yeah you're right, i forgot to remove my docker volume after making the change manually so i thought the migration worked 😅